airflow/ui/openapi-gen/queries/queries.ts

```diff
@@ -152,6 +152,10 @@ export const useDagServiceGetDags = <
         paused,
         tags,
       }) as TData,
+    staleTime: 5 * 60 * 1000,
+    refetchOnWindowFocus: false,
```

Comment:
Oh actually we can't have these changes in the autogenerated files. We can either adjust the defaults in the react query provider or pass the options when we call the hook in DagsList

Reply:
Sorry i didn't see this until just recently. I refactored this and i did a rebase and force push. Should be ready for review again.